target/linux/ar71xx/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -19,6 +19,9 @@ case "$board" in
 	archer-c60-v2)
 		echo $(macaddr_add $(mtd_get_mac_binary mac 8)  $(($PHYNBR - 1)) ) > /sys${DEVPATH}/macaddress
 		;;
+	tew-823dru)
+	        echo $(macaddr_add $(strings $(find_mtd_part mac) | head -n1 ) $(($PHYNBR + 1)) ) > /sys${DEVPATH}/macaddress
```

Comment:
@ppancha Sorry to bother you again, but I've just re-read the patch again and it seems to me, that it would be better to use `mtd_get_mac_ascii` or `mtd_get_mac_text` from `lib/functions/system.sh` instead of this strings/head fiddling. Would it be possible?

Reply:
Thanks for the feedback. Updated patch to use mtd_get_mac_text.